.github/workflows/release.yml

```diff
@@ -20,29 +20,30 @@ jobs:
     steps:
       -
         name: Checkout
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v3
       -
         name: Unshallow
         run: git fetch --prune --unshallow
       -
         name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.14
+          go-version: 1.16
       -
         name: Import GPG key
         id: import_gpg
-        uses: hashicorp/ghaction-import-gpg@v2.1.0
-        env:
+        uses: crazy-max/ghaction-import-gpg@v5
```

Comment:
FYI [deprecated notes](https://github.com/hashicorp/ghaction-import-gpg#warning-this-action-has-been-deprecated) on hashicorp gh workflow